azerothcore/azerothcore-wotlk
# Feat(Core/Config): Adjustable BG Respawn Timers

## CHANGES PROPOSED:
Adds option to `worldserver.conf` for setting the battleground respawn timer for players (time between resurrections while dead at the graveyard) and speed/berserker/rejuvenation buffs.

## ISSUES ADDRESSED:
- Closes #3551

## HOW TO TEST THE CHANGES:
Player respawn
1. Enter a battleground and wait for it to start.
2. Die and release spirit.
3. Check resurrection timer. 
(repeat as necessary)
4. Change "Battleground.PlayerRespawn" inside worldserver.conf
5. .reload config
6. Repeat steps 2-5.

Buff respawn
1. Enter a battleground and wait for it to start.
2. Take a buff and time how long it takes to respawn.
3. Change "Battleground.BuffRespawn" inside worldserver.conf
4. .reload config
5. Repeat steps 2-4.

## Target branch(es):

## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
